python/pyfunc-server/run.sh

```diff
@@ -13,6 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+sigterm_handler() {
+  echo "Terminating.."
+  kill -TERM "$server_pid"
+  wait "$server_pid"
+}
+
+sigint_handler() {
+  echo "Terminating.."
+  kill -INT "$server_pid"
+  wait "$server_pid"
+}
+
+trap sigterm_handler SIGTERM
```

Comment:
what is the purpose of trap here?

Reply:
The trap will allow catching signal that's received by bash and bash can forward it to the process via the handler.